### ISSUE TITLE
a11y - site setup - Date and Time - input fields - cms ui - readable labels

### ISSUE THREAD

**Issue by Wagner3UB** [plone/volto#5139]
**Describe the bug**

- None of the inputs have a proper label (aria-label / aria-labeledby)
- With a screen reader, all 3 inputs are read as "empty", but none are. 

**Checks:**

- [x] Proper label for inputs
- [x] Screen reader correct read the input and the fields names

**To Reproduce**

1. Go to "site setup" -> "Date and Time" 
2. With a screen reader + keyboard, go to the first input
3. The screen reader read "text field, empty" without focus
4. Pressing the spacebar, the dropdown menu will appear
5. It's possible to navigate using arrow down or arrow up 

**Expected behavior**
- The input field has to indicate its purpose to the screen reader, with a label or aria-labeledby
- The input field must indicate the select(s) value(s) to the screen-reader

**Screenshots**
<img width="1207" alt="Schermata 2023-08-28 alle 14 36 48" src="https://github.com/plone/volto/assets/60133113/a70a18e7-4bc8-49ca-b295-77777bd856b8">

<img width="1201" alt="Schermata 2023-08-28 alle 15 13 03" src="https://github.com/plone/volto/assets/60133113/fed3fed7-43bd-43bb-b4b0-99c5e343a677">

**Software (please complete the following information):**

Software (please complete the following information):
Volto Version 17.0.0-alpha.19
Plone Version 6.0.4

**Additional context**
These inputs have another pa11y problem, none of the "x" to close/cancel button works with the keyboard. I will open another issue for that specific problem.

**Comment by folix-01** [comment]
Solved here: https://github.com/plone/volto/pull/6332

**Comment by Wagner3UB** [comment]
Changed the PR description to mention and link the PR here.

**Comment by Wagner3UB** [comment]
> Solved here: #6332

@folix-01 Your PR doesn't solve all the problems mentioned in this issue, right? You addressed the labels, and that's good.

However, we still need to deal with the lack of visible focus. I will update the description to provide a clearer view of the remaining issues.

**Comment by folix-01** [comment]
> * visible focus

"visible focus" means :focus-visible?

**Comment by Wagner3UB** [comment]
> > * visible focus
> 
> "visible focus" means :focus-visible?

Not necessarily, you can implement focus using other CSS properties, such as :hover and :active. I think you need to check why the default browser focus wasn't appearing on those inputs, because all browsers have a default focus border for inputs.

**Comment by folix-01** [comment]
@Wagner3UB I edited the issue and moved the part of visible focus to another ticket, so we can have this one closed

**Comment by Wagner3UB** [comment]
This issue was reopened because not all problems have been solved.
